embed-api-v3.rst

```diff
@@ -0,0 +1,170 @@
+- Support for ``glossary`` / ``term`` and sphinxcontrib-bibtex.
+- Avoid showing the browser built-in tooltip for links that have the extension enabled.
+
+Finally, version 3 of the Embed API
+paves the way for supporting non-Sphinx projects in the future.
+
+Embedding content with sphinx-hoverxref
+---------------------------------------
+
+To use sphinx-hoverxref in your Read the Docs project,
+:doc:`declare it as part of your dependencies <guides/reproducible-builds>`:
+
+.. code-block::
+   :caption: requirements.txt
+   :emphasize-lines: 3
+
+   sphinx==4.3.1
+   sphinx_rtd_theme==1.0.0
+   sphinx-hoverxref==1.0.0
+
+And add it to your list of Sphinx extensions:
+
+.. code-block::
+   :caption: conf.py
+   :emphasize-lines: 3
+
+   extensions = [
+       # ...other extensions
+       "hoverxref.extension",
+   ]
+
+To enable the extension on all ``:ref:`` of your documentation,
+set the ``hoverxref_auto_ref`` to ``True``:
+
+.. code-block::
+   :caption: conf.py
+
+   hoverxref_auto_ref = True
+
+And you will start seeing tooltips on your internal references.
+Apart from ``:ref:`` roles, you can also enable tooltips on:
+
+- :ref:`Code objects from any Sphinx domain <hoverxref:Usage:Tooltip on Sphinx Domains>`,
+- :ref:`Glossary terms <hoverxref:Usage:Tooltip on glossary terms>`,
+- :ref:`BibTeX citations <hoverxref:Usage:Tooltip on sphinxcontrib-bibtex cites>`,
+- :ref:`Pages containing extra JavaScript like sphinx-tabs and
+  MathJax <hoverxref:Usage:Tooltip with content that needs extra rendering steps>`,
+- :ref:`Intersphinx references <hoverxref:Usage:Tooltip on intersphinx content>`,
+- :ref:`And custom objects! <hoverxref:Usage:Tooltip on custom object>`
+
+.. note::
+
+   sphinx-hoverxref includes the JavaScript embed client in the HTML assets,
+   but it is not yet available as a standalone library that can be reused
+   with standard frontend packaging tools.
+   If you would like to see this happening,
+   `let us know <https://github.com/readthedocs/sphinx-hoverxref/issues/>`_.
+
+Calling the Embed API directly
+------------------------------
+
+As explained in :ref:`our embedding
+guide <guides/embedding-content:calling the embed api directly>`,
+you can call the API directly from any HTTP client:
+
+.. code-block:: console
+
+   $ curl -s "https://readthedocs.org/api/v3/embed/\
+   > ?url=https://docs.readthedocs.io/en/latest/features.html\
+   > %23read-the-docs-features" | python -m json.tool
+   {
+     "url": "https://docs.readthedocs.io/en/latest/features.html#read-the-docs-features",
+     "fragment": "read-the-docs-features",
+     "content": "<div class=\"section\" id=\"read-the-docs-features\">\n<h1>Read the Docs ...",
+     "external": false
+   }
+
+Or visually explore the query in the `web interface`__ instead.
+
+.. __: https://readthedocs.org/api/v3/embed/?url=https://docs.readthedocs.io/en/latest/features.html%23read-the-docs-features
+
+The :ref:`reference documentation <api/v3:embed>`
```

Comment:
Are we not using a `readthedocs` prefix here?

Reply:
Good catch, changing.

Reply:
...although it worked without it thanks to Sphinx implicit rules :) (but better to add it)